### DISCUSSION TITLE
Adopt MySQL vs Postgres

### DISCUSSION THREAD

**Discussion by FlattusBlastus** [nextcloud/all-in-one#2202] [Ideas] [answered]
MySQL has become one of the most reliable, secure, and frankly the best database available outside of going to MS SQL or full on Oracle. With so many other linux products using it, I'd propose that Nextcloud AIO move to it as its official database of choice.

**Comment by szaimen** [comment, marked as answer]
Hi, we have no intentino to switch the database since Postgresql actually works much better with Nextcloud in our experience and is even faster.

**Reply by MrRinkana** [reply]
Would there be reason for the official nc docs to then recommend postgres? Currently mariadb is the recommended database according to the [system requirements](https://docs.nextcloud.com/server/stable/admin_manual/installation/system_requirements.html).

Are there more specific findings you could share about postgres working much better?
